crane-cloud/frontend
# Project Collaboration

# Description

- Project owners can invite collaborators to their projects under two roles, such as a member or administrator.
- Project owners can change roles for either collaborators [member/admin]
- Project owners can remove collaborators from their project [member/admin]
- Project [members/admins] can't view project management controls such as [update/delete] project

## Type of change


## Trello Ticket ID


## How Can This Been Tested?

Pull and run this branch locally, then visit the project settings page to see the Membership section.


## Checklist:


# Screenshots

![image](https://user-images.githubusercontent.com/63339234/198539359-f0af7170-5455-4c15-be91-8a9e0751f8ce.png)
